### DISCUSSION TITLE
Decouple the Admin Panel from Core API

### DISCUSSION THREAD

**Discussion by obezzad** [directus/directus#14020] [Feature Requests]
Hello Directus team and contributors,

I would like to know whether it's planned to decouple the core API from the admin panel, so we can customize the latter or create our own with the tech stack and design that suits us the most?

Thank you in advance

**Comment by br41nslug** [comment]
The API and App are already pretty much decoupled in most ways, as in everything the app does goes through the API. While they are bundled for ease of use you can disable the app using the `SERVE_APP=false` environment variable and build a custom app that suits your needs (the disabling of the app is not needed for this of course but will help prevent the confusion of having 2 management apps)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
